runtime/jcl/common/criu.cpp

```diff
@@ -64,6 +64,19 @@ Java_openj9_internal_criu_InternalCRIUSupport_isCRIUSupportEnabledImpl(JNIEnv *e
 	return res;
 }
 
+jboolean JNICALL
+Java_openj9_internal_criu_InternalCRIUSupport_isNonPortableRestoreModeImpl(JNIEnv *env, jclass unused)
```

Comment:
This is a JNI implementation that supports java native method `openj9.internal.criu.InternalCRIUSupport.isNonPortableRestoreModeImpl()` which is not in this PR. I think only `J9InternalFunctions` `isNonPortableRestoreMode` is needed as you already implemented. In that case (java method not required), `Java_openj9_internal_criu_InternalCRIUSupport_isNonPortableRestoreModeImpl` related change should be removed.

Reply:
Agreed, I've dropped this patch.